pkg/api/v1/types.go

```diff
@@ -354,6 +371,11 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
I'll need to carefully review that entire thread to understand it fully.  Is this just a naming thing (from Phase to Condition) or will it be altogether different?